moped-etl/prefect/tasks/api.py

```diff
@@ -99,7 +101,7 @@ def create_zappa_config(basename, config_secret_arn):
                 "AWS_COGNITO_DYNAMO_SECRET_NAME": AWS_STAGING_DYNAMO_DB_ENCRYPT_KEY_SECRET_NAME,
                 # Look at Moped API events.py to see how this key is used
                 "MOPED_API_HASURA_APIKEY": MOPED_API_HASURA_APIKEY,
-                "MOPED_API_HASURA_SQS_URL": MOPED_API_HASURA_SQS_URL,
+                "MOPED_API_HASURA_SQS_URL": create_activity_log_queue_url(basename),
```

Comment:
We can get away with creating the SQS enpoint from the `basename` so it doesn't rely on the activity log deployment.